k8s/k8s.bzl

```diff
@@ -80,9 +80,9 @@ py_library(
     if "io_bazel_rules_docker" not in excludes:
         http_archive(
             name = "io_bazel_rules_docker",
-            sha256 = "6287241e033d247e9da5ff705dd6ef526bac39ae82f3d17de1b69f8cb313f9cd",
-            strip_prefix = "rules_docker-0.14.3",
```

Comment:
0.14.3 the latest release, please work with rules_docker to get a new release out that includes any fixes you want.

https://github.com/bazelbuild/rules_docker/releases